projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos_eth/aws_demos_eth.uvproj

```diff
@@ -1546,6 +1546,11 @@
 							<FileType>1</FileType>
 							<FilePath>../../../../../demos/coreMQTT/mqtt_demo_plaintext.c</FilePath>
 						</File>
+						<File>
+							<FileName>mqtt_demo_mutual_auth.c</FileName>
+							<FileType>1</FileType>
+							<FilePath>../../../../../demos/coreMQTT/mqtt_demo_mutual_auth.c</FilePath>
+						</File>
 					</Files>
```

Comment:
Sorry there is on more place above the mqtt_demo_plaintext_config.h file tag:
```
						<File>
							<FileName>mqtt_demo_mutual_auth_config.h</FileName>
							<FileType>5</FileType>
							<FilePath>../../../../../vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/config_files/mqtt_demo_mutual_auth_config.h</FilePath>
						</File>
```

Reply:
Added